modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -55,15 +55,33 @@ If you use SSH key-based authentication, you must manage the key with an SSH age
 # subscription-manager attach --pool=<pool_id>
 ----
 
-. Enable the repositories required by {product-title} {product-version}:
+. Enable the repositories required by {product-title} {product-version}.
+
+.. For {op-system-base} 7 nodes, you must enable the following repositories:
 +
 [source,terminal]
 ----
 # subscription-manager repos \
     --enable="rhel-7-server-rpms" \
     --enable="rhel-7-server-extras-rpms" \
     --enable="rhel-7-server-ansible-2.9-rpms" \
-    --enable="rhel-7-server-ose-4.8-rpms"
+    --enable="rhel-7-server-ose-4.9-rpms"
+----
++
+[NOTE]
+====
+Use of {op-system-base} 7 nodes is deprecated and planned for removal in a future release of {product-title} 4.
+====
+
+.. For {op-system-base} 8 nodes, you must enable the following repositories:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-8-for-x86_64-baseos-rpms" \
+    --enable="rhel-8-for-x86_64-appstream-rpms" \
+    --enable="ansible-2.9-for-rhel-8-x86_64-rpms" \
+    --enable="rhocp-4.9-for-rhel-8-x86_64-rpms"
```

Comment:
I just realized we don't have openshift-ansible el8 packages, so maybe we need to remove this part of installing openshift-ansible on RHEL-8 host? @mtnbikenc could you help to confirm?

Reply:
That is a good point.  We don't build openshift-ansible for RHEL8.  In the effort to support RHEL8 workers, we didn't include any tasks to support RHEL8 control hosts.

Reply:
@gpei @mtnbikenc what changes will this require? Just removing that `ansible-2.9-for-rhel-8-x86_64-rpms` repo from the list above? Or more substantial changes? There are quite a few Ansible references in the [related assembly](https://deploy-preview-35928--osdocs.netlify.app/openshift-enterprise/latest/machine_management/adding-rhel-compute.html).

Reply:
There are two things in play here.  One, RHEL 8 worker node support.  Two, where the playbooks can be run.  This section is about where the playbooks are run, or the 'control host.'  The additions to this section could be dropped and only reference the change in repo from 4.8 to 4.9 for the RHEL7 control host.  Once we have RHEL8 packages built and test the install using RHEL8 control host, we can come back and add the RHEL8 repos as listed above.  When I originally suggested the changes by adding the RHEL8 repos, I had not confirmed the availability of RHEL8 openshift-ansible packages.